util/mem.h

```diff
@@ -5,6 +5,12 @@
 #include <stddef.h>
 #include <stdbool.h>
 
+#define CHECK_NULL_PTR(ptr) \
+	do { \
+		if (!ptr) \
+			abort(); \
+	} while (false)
+
```

Comment:
I don't think we need this macro as API. The whole point if this would be that we nvme-cli aborts directly when the allocation fails. Nothing should call this macro.